src/routes/index.js

```diff
@@ -1,7 +1,6 @@
-import express from 'express';
 
-const router = express.Router();
+import usersRoute from './api/users';
 
-router.use('/api', require('./api'));
-
-module.exports = router;
```

Comment:
we dont need to remove this line too...

Reply:
@oyewoas  we're writing es6 and we need to use es6 standard, suggest to look more into this.
All I did there was trying to organize the file structure and imports

Reply:
> @oyewoas we're writing es6 and we need to use es6 standard, suggest to look more into this.
> All I did there was trying to organize the file structure and imports

yes thats nice, i believe you could have just changed those lines to ES6 format rather than deleting it.. for example   
```javascript
import express from 'express';
import api from './api';

const router = express.Router();

router.use('/api', api);

export default router;
```